mautic/mautic
# The Add Notes Button is always displayed (also on smaller screens)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


The add note button in the Contact Notes tab is hidden by input on the small screen. This PR fix this.

Before:
![image](https://user-images.githubusercontent.com/96085911/200260163-4c1a48a7-e282-46aa-a3a8-81111a2a74f8.png)
![image](https://user-images.githubusercontent.com/96085911/200259768-02413eed-c966-4a14-bb06-2afe6452d063.png)
Now:
![image](https://user-images.githubusercontent.com/96085911/200260316-6b320704-2cda-4fd5-9b7c-8c1abc2e4807.png)
![image](https://user-images.githubusercontent.com/96085911/200259648-76c213d9-50b1-495c-9e26-07cb58b54bb8.png)

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Change the screen size, make it small
3. The add note button is fully visible
